README.md

```diff
@@ -53,8 +29,6 @@ After you're finished please remove all the comments and instructions!
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
 # 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
[Optional] You have done great implementing the new readme template. I suggest that you replace `[your_project_name]` with your actual project name. The text is only a placeholder and guide provided by the template.

Reply:
Thanks for the comment 🙏🏾 . I will quickly fix it 💯